cost/turbonomics/delete_unattached_volumes/gcp/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## v0.5
+
+- Added `Provisioned IOPs` incident field.
+- Added `Last VM` incident field.
+- Added `Action State` incident field.
+- Added `Disruptiveness` incident field.
+- Added `Reversibility` incident field.
+- Added `System Details URL` incident field.
+- Renamed `Attachment State` incident field to `State`.
+- Renamed `Created Time` incident field to `Action Created Time`.
+- Renamed `Unused Days` parameter to `Days Unattached`.
+- Changed internal names of several incident fields to ensure that they are properly scraped for dashboards.
+- Removed hardcoded host on business units path.
```

Comment:
This specific item isn't clear. It should be modified so that it would make sense to someone using the policy who doesn't know how to code or know anything about the policy template language.

Reply:
Hello, @XOmniverse. I have changed the item description. Could you let me know if new description works better? Thank you!